apisix/plugins/google-cloud-logging.lua

```diff
@@ -43,7 +43,7 @@ local schema = {
                     default = "https://oauth2.googleapis.com/token"
                 },
                 -- https://developers.google.com/identity/protocols/oauth2/scopes#logging
-                scopes = {
+                scope = {
```

Comment:
why should this be renamed?

Reply:
According to OAuth2/OIDC rules, scope should not be used in the plural.
![image](https://github.com/user-attachments/assets/01ceef0b-745f-4848-b789-51028cbb0a59)

Reply:
@HuanXin-Chen 

So this means destructive updates right?

I'd suggest keeping `scopes` as an alias for `scope` for now, and documenting that it's deprecated. (That is, if `scopes` is used in the configuration, our code needs to rewrite it as `scope` to be passed to google_oauth)

That way users won't experience breakage issues when upgrading and the documentation will remind them to change this field.

Reply:
Good point, I'll make the changes tonight. Thank you, mentor.🫡

Reply:
Resolved.